packages/vuetify/src/components/VDataIterator/VDataFooter.ts

```diff
@@ -99,6 +99,8 @@ export default Vue.extend({
       let value = this.options.itemsPerPage
       const computedIPPO = this.computedItemsPerPageOptions
 
+      if (computedIPPO.length <= 1) return null
+
       if (
         computedIPPO.length > 0 &&
```

Comment:
Line 102 can only have 3 possible outcomes.

- IPPO has a length of 0: returns null
- IPPO has a length of 1: returns null
- IPPO has a length > 1: continues

If we continue, we have a value that is greater than 1 which means 105 will always be true.